scripts/src/client.ts

```diff
@@ -417,6 +438,24 @@ export class Client {
 		}
 	}
 
+	public async createIncomingTransferOrder(walletAddress: string, appId: string, title: string, description: string, memo: string): Promise<OpenOrder> {
```

Comment:
The return from incoming/orders is a regualr `Order`, not `OpenOrder`

Reply:
done.